### PR TITLE
WindowsPB: Prevent Installation Of Firefox & Bump Installation version of Thunderbird

### LINKED ISSUE #3239: Investigate Removal Of FireFox & Thunderbird

### PR DESCRIPTION
**Pull request by steelhead31**
Fixes #3239 

Remove installation of firefox from the windows playbook due to security concerns, and restrict installation of Thunderbird to JCK machines. In addition bump the version of thunderbird to the current version ( 115.4 ), which currently has no known security issues.

##### Checklist

- [X] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [x] playbook changes run through [VPC](https://ci.adoptium.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptium.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)

### CODE REVIEW

**Comment by steelhead31** [comment]
VPC in progress: https://ci.adoptium.net/job/VagrantPlaybookCheck/1738/

**Review by gdams — APPROVED** [review]

**Review by tellison — APPROVED** [review]

File: ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@
     - CodesignCert
     - 7-Zip                       # Mostly extracting other prereqs :-)
     - cygwin
-    - Firefox
+#    - Firefox                    # Redundant No Longer Required
```
Comment (sxa) [inline ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml:53]: I'd suggest a straight removal of this

File: ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml [hunk 2]
```diff
@@ -90,7 +90,7 @@
       tags: [jenkins, adoptopenjdk]
       when: jenkins_secret is defined  # Only run if jenkins_secret is defined
     - role: Thunderbird
-      tags: [Thunderbird, jck, adoptopenjdk]
+      tags: jck
```
Comment (sxa) [inline ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml:93]: We don't routinely skip the `jck` tag on our runs AFAIK so I'm not sure this is the right approach.

Comment (sxa) [reply ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml:93]: Thunderbird is explicitly installed in https://github.com/temurin-compliance/infrastructure/blob/master/ansible/playbooks/JCK_Playbook/roles/thunderbird/tasks/main.yml so I don't believe it should be needed here in our main adoptium playbooks. @fredg02 Can you confirm please?

File: ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml (hunk 1 above)
Comment (steelhead31) [reply ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml]: Done.

File: ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml (hunk 2 above)
Comment (steelhead31) [reply ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml:92]: > We don't routinely skip the `jck` tag on our runs AFAIK so I'm not sure this is the right approach.

what do you think to me adding a variable, so the role is disabled by default, unless you override or change the variable as a solution to this?

Comment (sxa) [reply ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml:92]: At the moment I say remove it, as the jck machines install it via another mechanism anyway :-)
But would be good to get confirmation from Fred first

Comment (steelhead31) [reply ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml]: Removed, and issue logged with EF infra team for confirmation.. https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3934

**Pull request merged**